sdk/core/Azure.Core/src/RequestContext.cs

```diff
@@ -70,38 +70,14 @@ public RequestContext()
         /// Initializes a new instance of the <see cref="RequestContext"/> class using the given <see cref="ErrorOptions"/>.
         /// </summary>
         /// <param name="options"></param>
-        public static implicit operator RequestContext(ErrorOptions options) => new RequestContext { ErrorOptions = options };
+        public static implicit operator RequestContext(ErrorOptions options)
+            => new RequestContext { ErrorOptions = options };
 
         /// <summary>
         /// TBD.
         /// </summary>
         /// <param name="message"></param>
-        public override void Apply(PipelineMessage message)
-        {
-            if (message is not HttpMessage httpMessage)
-            {
-                throw new InvalidOperationException($"Unexpected message type: '{message.GetType()}'.");
-            }
-
-            Apply(httpMessage);
-        }
-
-        /// <summary>
-        /// TBD.
-        /// </summary>
-        /// <param name="message"></param>
-        public void Apply(HttpMessage message)
-        {
-            message.CancellationToken = CancellationToken;
-
-            // TODO: Note, we don't set response classifier here, but need to rethink this
-            // story e2e.
-
-            if (NetworkTimeout.HasValue)
-            {
-                ResponseBufferingPolicy.SetNetworkTimeout(message, NetworkTimeout.Value);
-            }
-        }
+        internal void Apply(HttpMessage message) => base.Apply(message);
```

Comment:
Figure out what this looks like now that RO.Apply is public again.  I think this can be cleaned up and also possibly tidy up the RO.Apply implementation around classifier precedence rules?

Reply:
Addressed.